server/src/main/scala/org/elasticmq/server/QueueConfigStore.scala

```diff
@@ -0,0 +1,61 @@
+package org.elasticmq.server
+
+import com.typesafe.config.ConfigRenderOptions
+import org.elasticmq.actor.queue.QueuePersister
+import org.elasticmq.server.config.ElasticMQServerConfig
+import org.elasticmq.{QueueMetadata, QueueData}
+import pureconfig.ConfigWriter
+import pureconfig.generic.auto._
+
+import java.io.PrintWriter
+import scala.collection.mutable
+
+case class QueueConfigStore(config: ElasticMQServerConfig) extends QueuePersister {
+
+  private val queues: mutable.Map[String, QueueData] = mutable.HashMap[String, QueueData]()
```

Comment:
that's a thread-unsafe structure - we can't use it like this

Reply:
another alternative to keeping the queues state copy - upon receiving a notification, read in the current state from the file, modify, write it back